pages/features.html

```diff
@@ -112,289 +142,65 @@
     <div style="clear: both;"></div>
 
     <div class="gapfiller">
-      <H1 style="margin-left: 30px;">{% trans "Designer Skins" %}</H1>
+      <h1 style="margin-left: 30px;">{% trans "Designer Skins" %}</h1>
       <center>
-        <img width="800px" src="{% static '/static/images/skins-2.0.png' %}">
-        <p>{% trans "Includes 3 exclusive designer skins with configurable layouts" %}.</p>
+        <img width="800px" src="{% static '/static/images/skins.png' %}">
+        <p>{% trans "Three exclusive designer skins with configurable layouts" %}</p>
```

Comment:
Good idea. It [likely will](https://github.com/mixxxdj/mixxx/pull/1151) for 2.1.